ansible_ai_connect/ai/api/permissions.py

```diff
@@ -64,6 +94,12 @@ def has_permission(self, request, view):
         if user.rh_user_has_seat is True:
             return True
```

Comment:
You probably want to change this to `CONTINUE` for consistency?

L#93 and L#95 (GitHub won't let me comment on L#93)